src/meta/src/barrier/mod.rs

```diff
@@ -1125,16 +1128,23 @@ impl GlobalBarrierManagerContext {
                     node_need_collect,
```

Comment:
Is it possible to manage the collect handler in rpc manager as well? For example maintain a `map<epoch, handle>` in it and remove it on complete or abort them all on recovery. It is also acceptable to remain unchanged.

Reply:
Yes it's possible, but this PR is a temporary version of a final one. When we implement rpc as a bidirection stream between CN and meta, there is no need to spawn a task for rpc. So I think we can remain unchanged now.